modules/op-speccancel-pipeline-run-using-pipelines-as-code.adoc

```diff
@@ -26,10 +26,12 @@ annotations:
 # ...
 ----
 
+:FeatureName: Automatic cancellation-in-progress of pipeline runs
+include::snippets/technology-preview.adoc[]
```

Comment:
I am not sure where this assembly is used in the docs. Cannot verify that the snippet appears correctly .